docs/events_and_streams.md

```diff
@@ -0,0 +1,44 @@
+# The description of Event and Stream definitions
+
+`pg_eventstore` provides classes to manipulate the data in the database. The most important are:
```

Comment:
Misleading, you do not manipulate, you just append. So maybe better:

`pg_eventstore` provides classes to prepare the events to be inserted into the eventstore. The most important are:

Reply:
Yeah, thanks.